javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,104 @@
+let userName = "";
+let todos = [] ;
+
+function getReply(command) {
+if (command.startsWith("Hello my name is")) {
+let name = command.split(" ").pop();
+if (userName) return `You've already introduced yourself as ${userName}.`;
+userName = name;
+return `Nice to meet you, ${userName}`;
+}
+
+if (command === "What is my name?") {
+return userName ? `Your name is ${userName}` : "I don't know your name yet.";
+}
+
+if (command.startsWith("Add") && command.endsWith("to my todo")) {
+let task = command.slice(4, -10). trim();
+todos.push(task);
+return `${task} added to your todo`;
+}
+
+
+
+if (command.startsWith("Remove") && command.endsWith("from my todo")) {
+    let task = command.slice(7, -13). trim(); // Extract tasks from the command
+let index = todos.indexOf(task);
+if (index !== -1) {
+todos.splice(index, 1);
+return `Removed ${task} from your todo`;
+}
+return `${task} is not in your to do list.`;
+}
+
+if (command === "What is on my todo?") {
+return todos.length ? `You have ${todos.length} todos: ${todos.join (", ")}` : "Your todo list is empty."}
+
+if (command === "What day is it today?") {
+return new Date ().toDateString();
+} 
```

Comment:
Per the homework, you should format your date as expected. 
(_Should respond with the date in a human readable format. E.g. if today is 30/8/2019 then it should respond with 30. of August 2019_)

Hint: [toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)